cli/launchpad/org.go

```diff
@@ -0,0 +1,86 @@
+package launchpad
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+type orgYAML struct {
+	headerYAML `yaml:",inline"`
+	Spec       orgSpecYAML `yaml:"spec"`
+}
+
+func (o *orgYAML) enroll(e *eval) error { return o.Spec.enroll(e) }
```

Comment:
These wrapping funcs definitely smell like unnecessary code to me. Let's just operate entirely on the top level (or entirely on the specs).